pygeo/mphys/mphys_dvgeo.py

```diff
@@ -125,7 +128,39 @@ def compute(self, inputs, outputs):
         # next time the jacvec product routine is called
         self.update_jac = True
 
-    def nom_addChild(self, ffd_file, DVGeoName=None, childName=None):
+    def nom_addComponent(self, comp, ffd_file, triMesh, DVGeoName=None):
```

Comment:
Do we need this pass through function to modify anything on the OM side or could everything be handled with DVGeo calls in a runscript?

Reply:
I thought about this too and debated what to do. The user would have to import DVGeo and make a DVGeo object to then add it to the DVGeoMulti object. It would work and I can change it to that, but this call streamlines the process. I agree that if the goal is to include only things that actually modify OM in the MPhys wrapper then we should change this.

Reply:
That's fair, I think in this case it's cleaner to handle it with this function.